website/docs/r/default_vpc.html.markdown

```diff
@@ -43,6 +43,7 @@ The arguments of an `aws_default_vpc` differ slightly from those of [`aws_vpc`](
 The following additional arguments are supported:
 
 * `force_destroy` - (Optional) Whether destroying the resource deletes the default VPC. Default: `false`
+* `tags` - (Optional) Key-value mapping of resource tags. If configured with a provider [`default_tags` configuration block](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#default_tags-configuration-block) present, tags with matching keys will overwrite those defined at the provider-level.
```

Comment:
similarly, `tags` is documented on the vpc resource as well:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc#tags

Reply:
@jar-b Good catch. I've removed the change.